en/websites_privacy_notice.md

```diff
@@ -1,26 +1,27 @@
 
 * **User Feedback**: You can provide feedback to us on our products and services on webpages like [input.mozilla.org](https://input.mozilla.org/), through an in-product experience, or through channels such as email, Bugzilla, Matrix, a social media account, our Get Involved page, or through a group like Student Ambassadors. Please minimize the personal information you choose to share on these forums because your comments may be accessible to the public.
 
-* **Accounts & Subscriptions**: Some websites, for example AMO and MDN require account creation to submit developer content or submit user reviews or comments. For account management we use [Firefox Accounts](https://www.mozilla.org/privacy/firefox/), [GitHub](https://help.github.com/en/github/site-policy/github-privacy-statement#our-use-of-cookies-and-tracking), or custom systems; see [here](https://support.mozilla.org/kb/managing-account-data) to manage account data. You may periodically receive emails in connection with your account or through  subscriptions. Our email management vendors are [SalesForce Marketing Cloud](https://www.marketingcloud.com/privacy-policy/website-privacy-statement/), [Send in Blue](https://www.sendinblue.com/legal/privacypolicy/), [Mailchimp](https://mailchimp.com/legal/privacy/), or [Acoustic](https://acoustic.com/privacy-notice/) and you can unsubscribe using the link at the bottom of the email. 
+* **Accounts & Subscriptions**: Some websites, for example [Add-Ons for Firefox](https://addons.mozilla.org/firefox/), [Relay](https://relay.firefox.com/), [Monitor](https://monitor.firefox.com/), and [MDN](https://developer.mozilla.org/), require account creation. For account management we use [Mozilla accounts](https://www.mozilla.org/privacy/mozilla-accounts/), [GitHub](https://help.github.com/github/site-policy/github-privacy-statement#our-use-of-cookies-and-tracking), or custom systems; [learn more](https://support.mozilla.org/kb/managing-account-data) about how to manage your Mozilla account data. You may periodically receive emails in connection with your account or through subscriptions. Our email management vendors are [SalesForce Marketing Cloud](https://www.marketingcloud.com/privacy-policy/website-privacy-statement/), [Amazon Simple Email Service](https://aws.amazon.com/privacy/), [Mailchimp](https://mailchimp.com/legal/privacy/), [SocketLabs](https://www.socketlabs.com/legal/service-privacy/), [Campaign Monitor](https://meetmarigold.com/privacy-notices) or [Acoustic](https://acoustic.com/privacy-notice/), and you can unsubscribe using the link at the bottom of the relevant email.
```

Comment:
```suggestion
* **Accounts & Subscriptions**: Some websites, for example [Add-Ons for Firefox](https://addons.mozilla.org/firefox/), [Relay](https://relay.firefox.com/), [Monitor](https://monitor.firefox.com/), and [MDN](https://developer.mozilla.org/), require account creation. For account management we use [Mozilla accounts](https://www.mozilla.org/privacy/mozilla-accounts/), [GitHub](https://support.github.com/#our-use-of-cookies-and-tracking), or custom systems; [learn more](https://support.mozilla.org/kb/managing-account-data) about how to manage your Mozilla account data. You may periodically receive emails in connection with your account or through subscriptions. Our email management vendors are [SalesForce Marketing Cloud](https://www.salesforce.com/company/privacy/), [Amazon Simple Email Service](https://aws.amazon.com/privacy/), [Mailchimp](https://www.intuit.com/privacy/statement/), [SocketLabs](https://www.socketlabs.com/legal/service-privacy/), [Campaign Monitor](https://meetmarigold.com/privacy-notices) or [Acoustic](https://acoustic.com/privacy-notice/), and you can unsubscribe using the link at the bottom of the relevant email.
```

A few of these links redirect (GitHub, Salesforce, MailChimp). Worth updating them?